src/index.js

```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import "./index.css";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
-import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
we are using bootstrap, please import "bootstrap/dist/css/bootstrap.min.css";

Reply:
@ayshesaflo please fix this as Deniz wrote